Tests/CommandsTests/PackageCommandTests.swift

```diff
@@ -4106,11 +4106,12 @@ class PackageCommandSwiftBuildTests: PackageCommandTestCase {
         throw XCTSkip("SWBINTTODO: Build plan is not currently supported")
     }
 
+#if !os(macOS)
```

Comment:
question: why are we not executing this test on macOS?

Reply:
it was a vestige of an earlier diff, dropped it